scripts/test_restore_points.sh

```diff
@@ -297,6 +297,11 @@ if [ ${PG_VERSION_MAJOR} -lt 13 ]; then
   exit 1
 fi
 
+if [ ${PG_VERSION_MAJOR} -ge 16 ]; then
```

Comment:
The check is performed in the shell script and not excluded from the build matrix in the workflow since matrix exclusion would require the exact PG version and needs to be adjusted with every version upgrade. 

```
  exclude:
    - pg: "16.0"
```
Here, we can perform the test based on the PG major version.